extensions/parquet/base/src/main/java/io/deephaven/parquet/base/ColumnChunkReaderImpl.java

```diff
@@ -177,14 +174,16 @@ private Dictionary readDictionary(SeekableByteChannel file, long dictionaryPageO
         }
         final DictionaryPageHeader dictHeader = pageHeader.getDictionary_page_header();
 
-        BytesInput payload = BytesInput.from(Helpers.readFully(file, pageHeader.compressed_page_size));
-        if (decompressor != null) {
-            payload = decompressor.get().decompress(payload, pageHeader.uncompressed_page_size);
+        final BytesInput payload;
+        int compressedPageSize = pageHeader.getCompressed_page_size();
+        if (decompressor != null && compressedPageSize > 0) {
```

Comment:
If we make it so `decompressor` can never be `null` (and I think we should), we can probably simply this if.

Reply:
Fixed - zero is a real use case apparently, I've made this explicitly an empty BytesInput.